client/app/components/message-list.coffee

```diff
@@ -51,7 +51,7 @@ module.exports = MessageList = React.createClass
         account   = AccountStore.getByID accountID
         role = AccountStore.getMailboxRole account, mailboxID
         settings = SettingsStore.get()
-        displayConvs = !!(settings.get('displayConversations') and
+        displayConvs = !!(settings.get('displayConversation') and
```

Comment:
Feel like we might as well drop the settings.displayConversation (cc @m4dz for refactoring)